examples/azure/terraform-azure-synapse-example/variables.tf

```
@@ -0,0 +1,67 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# ENVIRONMENT VARIABLES
+# Define these secrets as environment variables
+# ---------------------------------------------------------------------------------------------------------------------
+
+# ARM_CLIENT_ID
+# ARM_CLIENT_SECRET
+# ARM_SUBSCRIPTION_ID
+# ARM_TENANT_ID
+
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# You must provide a value for each of these parameters.
+# ---------------------------------------------------------------------------------------------------------------------
+
+# ---------------------------------------------------------------------------------------------------------------------
+# OPTIONAL PARAMETERS
+# These parameters have reasonable defaults.
+# ---------------------------------------------------------------------------------------------------------------------
+
+variable "location" {
+  description = "The supported azure location where the resource exists"
+  type        = string
+  default     = "West US2"
+}
+
+variable "storage_account_kind" {
+  description = "The kind of storage account to set"
+  type        = string
+  default     = "StorageV2"
+}
+
+variable "storage_account_tier" {
+  description = "The tier of storage account to set"
+  type        = string
+  default     = "Standard"
+}
+
+variable "storage_replication_type" {
+  description = "The replication type of storage account to set"
+  type        = string
+  default     = "GRS"
+}
+
+variable "synapse_sql_user" {
+  description = "The sql pool user password for synapse"
+  type        = string
+  default     = "sqladminuser"
+}
+
+variable "sy_sqlpool_sku_name" {
```

update name to avoid obberviations for both variables sy_sqlpool_sku_name, and sy_sqlpool_create_mode
```suggestion
variable "synapse_sqlpool_sku_name" 
```

---

Thanks @HadwaAbdelhalem updated